compute_sdk/globus_compute_sdk/sdk/executor.py

```diff
@@ -1300,7 +1300,7 @@ def run(self):
                     self._open_futures_empty.set()
         log.debug("%r AMQP thread complete.", self)
 
-    def shutdown(self, wait=True, *, cancel_futures=False):
+    def shutdown(self, wait=True, *, cancel_futures=False) -> None:
```

Comment:
I'm not clear on the motivation for this hunk.

For some background, our Executor is [a subclass of Python's Executor](https://github.com/globus/globus-compute/blob/2.30.1/compute_sdk/globus_compute_sdk/sdk/executor.py#L117C1-L123C16):

```python
class Executor(concurrent.futures.Executor):
    """
    Extend Python's |Executor|_ base class for Globus Compute's purposes.

    .. |Executor| replace:: ``Executor``
    .. _Executor: https://docs.python.org/3/library/concurrent.futures.html#executor-objects
    """  # noqa
...
```

and [during the original ~implementation~ _refactor_ of the Executor](https://github.com/globus/globus-compute/pull/942), one of the main goals was to move to conformity with the super class' interface.